R/help/rdToHtml.R

```diff
@@ -0,0 +1,21 @@
+
+#' Converts an .Rd file to HTML (output is printed to stdout)
+#'
+#' Execute this with the following trailing commandline args:
+#'   1. path of an .Rd file
+#'   2. name of the package
+#'   3. version string of the package
+#'   4. root dir of the package
+
+e <- tools::loadPkgRdMacros(base::commandArgs(TRUE)[4])
```

Comment:
The value of `base::commandArgs(TRUE)` might be better stored here rather than being called multiple times.